x/interchainstaking/keeper/callbacks.go

```diff
@@ -253,6 +255,30 @@ func DepositIntervalCallback(k *Keeper, ctx sdk.Context, args []byte, query icqt
 	return nil
 }
 
+func SigningInfoCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes.Query) error {
+	zone, found := k.GetZone(ctx, query.GetChainId())
+	if !found {
+		return fmt.Errorf("no registered zone for chain id: %s", zone.ChainId)
+	}
+
+	k.Logger(ctx).Debug("Validator signing info callback", "zone", zone.ChainId)
+
+	valSigningInfo := slashingtypes.ValidatorSigningInfo{}
+	if len(args) == 0 {
+		return errors.New("attempted to unmarshal zero length byte slice (10)")
+	}
+	err := k.cdc.Unmarshal(args, &valSigningInfo)
+	if err != nil {
+		return err
+	}
+	if valSigningInfo.Tombstoned {
+		k.Logger(ctx).Error("Tombstoned validator found", "valoper", valSigningInfo.Address)
+		return fmt.Errorf("%q on chainID: %q was found to already have been tombstoned", valSigningInfo.Address, zone.ChainId)
```

Comment:
As discussed on the call, we need to not error here, but instead, update the validator record and return nil (finding that a validator is tombstoned is considered 'success' because we can update our records to reflect that).

Will almost certainly need to update the validator record to store the consensus key of the validator, in order to later lookup the validator my consensus key.